Tests/SwiftlyTests/HTTPClientTests.swift

```diff
@@ -66,15 +77,17 @@ final class HTTPClientTests: SwiftlyTests {
                 // GIVEN: we have a swiftly http client with swift.org metadata capability
                 // WHEN: we ask for the first five releases of a supported platform in a supported arch
                 let releases = try await SwiftlyCore.httpClient.getReleaseToolchains(platform: platform, arch: arch, limit: 5)
-                // THEN: we get five releases
-                XCTAssertEqual(5, releases.count)
+                // THEN: we get at least 1 release
+                XCTAssertTrue(1 <= releases.count)
+
+                if newPlatforms.contains(platform) { continue } // Newer distros don't have main snapshots yet
```

Comment:
How will we eventually know to remove this for proper testing? Seems like the kind of thing I'd forget 😄

Reply:
Integration tests that are in the middle of the test pyramid like this a tricky, but necessary.

The trouble with mocking the responses is that we won't know if the API might change in some way that breaks swiftly. I think that we want to catch obvious integration problems between the swift.org API and what we can potentially request from it. If that means that we don't fully cover all possibilities with all supported platforms I think that's ok. This was a good test when I started working on the patch to verify that the platform definition strings were correct from the standpoint of the swift.org API.